docs/maintenance.md

```diff
@@ -0,0 +1,49 @@
+# Maintenance mode of storage nodes
+
+## Overview
+
+Storage nodes go into maintenance mode (MM) while internal procedures are required
+to troubleshoot storage service issues. Data on MM nodes MAY be temporarily
+unavailable. Otherwise, such nodes do not differ from fully functional ones.
+Nodes go into MM on their own.
+
+## Reflection in the network map
+
+To globally notify the network about the start of maintenance procedures, node
+MAY set the corresponding state in the network map. The ability to set MM state
+is determined by the global network setting. If MM is disallowed at the moment,
+node will not be able to set this state. Setting change is not retroactive.
+
+To check the possibility of MM-switching at the moment, exec:
+```shell
+$ neofs-cli netmap netinfo
+...
+Maintenance mode allowed: true
+...
+```
+
+To switch the node to MM, exec:
+```shell
+$ control set-status --status maintenance
+```
+
+To stop the maintenance, use same command with any other supported state.
```

Comment:
```suggestion
To stop the maintenance, use the same command but with any other supported state.
```

Reply:
At least for the latest release, I am not able to see "Maintenance mode allowed" line while the MM is active.

It would be great to see "Maintenance mode allowed: true" while the MM is active and hide that line while we are in usual state.

Reply:
This setting didn't exist in previous release. I tested `netinfo` with `neofs-cli@v0.32.0`:
```
$ neofs-adm morph set-config MaintenanceModeAllowed=true
$ neofs-cli netmap netinfo
...
NeoFS network configuration (other)
  MaintenanceModeAllowed: 01
  SystemDNS: XXX
$ neofs-adm morph set-config MaintenanceModeAllowed=true
$ neofs-cli netmap netinfo
...
NeoFS network configuration (other)
  SystemDNS: XXX
$
```
Do you expect output like this?

Reply:
I have tested and it seems those commands work fine for me. Could you please tell what exact "set-config MaintenanceModeAllowed=true" command affects? Only for displaying "Maintenance mode allowed: true" line?

Reply:
@vkarak1 the command changes config value in the running NeoFS network. From other side, `netinfo` shows current settings in the network.